Binance Detect Moonings.py

```diff
@@ -360,9 +362,15 @@ def sell_coins():
 
     for coin in list(coins_bought):
         # define stop loss and take profit
-        TP = float(coins_bought[coin]['bought_at']) + (float(coins_bought[coin]['bought_at']) * coins_bought[coin]['take_profit']) / 100
-        SL = float(coins_bought[coin]['bought_at']) + (float(coins_bought[coin]['bought_at']) * coins_bought[coin]['stop_loss']) / 100
-
+        try:
+            TP = float(coins_bought[coin]['bought_at']) + (
+                    float(coins_bought[coin]['bought_at']) * coins_bought[coin]['take_profit']) / 100
+            SL = float(coins_bought[coin]['bought_at']) + (
+                    float(coins_bought[coin]['bought_at']) * coins_bought[coin]['stop_loss']) / 100
+        # When an older version of the script is being executed
+        except KeyError:
+            TP = float(coins_bought[coin]['bought_at']) + (float(coins_bought[coin]['bought_at'])) / 100
+            SL = float(coins_bought[coin]['bought_at']) + (float(coins_bought[coin]['bought_at'])) / 100
```

Comment:
Wouldn't be a good idea to put `coins_bought[coin]['bought_at']` into a variable to avoid repeating it too much?
Code might be cleaner